src/Parsers/ASTQueryWithOutput.h

```diff
@@ -16,6 +16,7 @@ class ASTQueryWithOutput : public IAST
 public:
     ASTPtr out_file;
     bool is_into_outfile_with_stdout;
+    bool is_outfile_append;
```

Comment:
Also this looks unsafe - it leaves the value non initialized, and the tests passed (w/o `APPEND`) only by pure luck.

Reply:
Fixed in #50216